server/context.go

```diff
@@ -138,6 +138,26 @@ func (s *SessionManager) SetDB(conn *mysql.Conn, db string) error {
 	return nil
 }
 
+// Iter iterates over the active sessions and executes the specified callback function on each one.
+func (s *SessionManager) Iter(f func(session sql.Session) (stop bool, err error)) error {
+	s.mu.Lock()
```

Comment:
Make a note about why this is necessary (prevent errors about iterating over a map while mutating it)